server/src/subscriber/subscriber.controller.ts

```diff
@@ -122,4 +122,24 @@ export class SubscriberController {
 
     response.send(updatedContact);
   }
+
+  @Post("/subscribers/:email/modify")
+  async modifySubscriptions(@Param() params, @Res() response) {
+    const existingUser = await this.subscriberService.findByEmail(params.email);
+    if (existingUser.code === 404) {
+      response.status(404).send({
+        error: "User not found."
+      })
+      return;
+    }
+
+    const userId = existingUser['1'].result[params.email].contact.custom_fields[`zap_${this.sendgridEnvironment}_id`];
+
+    // Send the modify email
+    await this.subscriberService.sendModifySubscriptionEmail(params.email, this.sendgridEnvironment, userId);
+    response.status(201).send({
```

Comment:
Sending this since it's a `POST` and also because the request hangs without a response. Not sure if there's a better way.